templates/medical-bill/templates/main.tpl

```diff
@@ -0,0 +1,282 @@
+            {{template "simple-paragraph" dict "Margin" "0" "LineHeight" 1 "Font" "helvetica" "FontSize" 10 "TextColor" "#000000" "Text" "For Billing inquries: 123 - 456 - 7890\nPatent Name: Sample Patent"}}
+        </division>
+    </table-cell>
+
+    {{with .bill.Guarantor}}
+        {{template "table-cell-paragraph" (extendDict $props "Colspan" 1 "Rowspan" 1 "BorderBottomSize" 0 "Margin" "0" "LineHeight" 1.5 "Font" "helvetica-bold" "FontSize" 12 "Text" (printf "%s\n%s\n%s" .Name .Address1 .Address2)) }}
+    {{end}}
+    {{template "table-cell-paragraph" (extendDict $props "Margin" "0" "LineHeight" "1.5" "Font" "helvetica-bold" "FontSize" 12 "Text" (printf "%s\n%s\n%s" .institution.name .institution.address1 .institution.address2)) }}
+</table>
+
+<page-break></page-break>
+
+{{template "simple-paragraph" dict "Margin" "0 0 5 0" "LineHeight" 1.0 "Font" "helvetica-bold" "FontSize" 16 "TextColor" "#000000" "Text" "The Sample Medical Center financial assistance policy plain language summary"}}
+{{template "simple-paragraph" dict "Margin" "0 0 10 0" "LineHeight" 1.0 "Font" "helvetica" "FontSize" 12 "TextColor" "#000000" "Text" "Sample Medical Center offers financial assistance to eligible patients who are uninsured, underinsured, and ineligible for a government health care program, or who are otherwise unable to pay for medically necessary care based on their individual financial situation.\nPatients seeking financial assistance must apply for the program, which is summarized below."}}
+
+{{template "simple-paragraph" dict "Margin" "5 0 5 0" "LineHeight" 1.0 "Font" "helvetica-bold" "FontSize" 14 "TextColor" "#000000" "Text" "Eligible Services"}}
+{{template "simple-paragraph" dict "Margin" "0 0 10 0" "LineHeight" 1.0 "Font" "helvetica" "FontSize" 12 "TextColor" "#000000" "Text" "Eligible services include emergent or medically necessary services provided by the Hospital. Eligible patients include all patients who submit a financial assistance application (including requested documentation) and are determined to be eligible for financial assistance by the Patient Financial Services Department."}}
+
+{{template "simple-paragraph" dict "Margin" "5 0 5 0" "LineHeight" 1.0 "Font" "helvetica-bold" "FontSize" 14 "TextColor" "#000000" "Text" "How to Apply"}}
+{{template "simple-paragraph" dict "Margin" "0 0 10 0" "LineHeight" 1.0 "Font" "helvetica" "FontSize" 12 "TextColor" "#000000" "Text" "Financial Assistance applications may be obtained/completed/submitted as follows:"}}
+
+<list indent="5">
+    <list-marker font="helvetica" font-size="12" color="#000000">• </list-marker>
+    <list-item>
+        <paragraph>
+            <text-chunk font="helvetica" font-size="12" color="#000000">Obtain an application at The Sample Medical Center's Patient Financial Services Department located at Main Street 123</text-chunk>
```

Comment:
`helvetica` is the default font for text chunks and `#000000` is the default color. 
There should be no need to specify these attributes in this case.